banzai/mosaic.py

```diff
@@ -40,6 +46,16 @@ def do_stage(self, image):
         return image
 
 
+def get_windowed_mosaic_slices(detsec, xmin, ymin, x_binning, y_binning):
+    unbinned_slice = fits_utils.parse_region_keyword(detsec)
```

Comment:
Want to assert that  slice_width / binning == slice_width // binning ? That would be a safeguard against wrong detsec / binning config (although I would never do that)

Reply:
Given the complexity of the indexing, I'm a little nervous to enforce that restriction. In this case, I'm inclined to just let the error get logged if a DETSEC keyword is incorrect rather than trying to code around every edge case.